draft-ietf-anima-constrained-join-proxy.md

```diff
@@ -595,7 +598,23 @@ without violating MTU sizes.
 
 ### JPY Message Security
 
-The Join Proxy SHOULD encrypt the state data prior to wrapping it in a CBOR byte string in jpy_header. 
+Application or ecosystem standards adopting Join Proxy need to determine if there is the potential
+for attacks against stateless join proxies: Senders other than a trustworthy registrar sending
+packets to the Join Proxy with the source address of a trustworthy registrar. In many well designed
+solutions, this attack vector can be excluded. For example, in ANI networks, the ACP ({{RFC8994}})
```

Comment:
```suggestion
for attacks originating on the trusted side of the stateless join proxies: 
This would involve senders other than a trustworthy registrar sending
packets to the Join Proxy, in-personating the trusted registrar's source address. 
In many well designed solutions, this attack vector can be excluded because source addresses are verified.
For example, in ANI networks, the ACP ({{RFC8994}})
```

(ps: New Line for New Sentence makes these changes easier)